src/components/views/rooms/RoomSettings.js

```diff
@@ -55,9 +55,20 @@ module.exports = React.createClass({
             power_levels_changed: false,
             tags_changed: false,
             tags: tags,
-            areNotifsMuted: areNotifsMuted
+            areNotifsMuted: areNotifsMuted,
```

Comment:
It would be polite to put `isRoomPublished` here, so we don't have to read the entire file to find out what our state vars are :)